python/tvm/testing/utils.py

```diff
@@ -2081,3 +2081,28 @@ def pprint(name, obj):
                 f"or an instance of `tvm.tir.PrimFunc`.  "
                 f"Instead, received {type(expected)}."
             )
+
+
+class _control_span_filling:
+    def __init__(self, on=True):
+        self._old_state = os.environ["TVM_SPANFILLING"] if "TVM_SPANFILLING" in os.environ else None
```

Comment:
Because we can command like the following way to get rid of span without modifying the code. It would be a bit more convenient. 
```bash
TVM_SPANFILLING=0 python ${your_program.py}
````

Reply:
right--sorry perhaps I should have commented on _should_fill_span in python/tvm/relay/frontend/common.py. what I mean is--could we consider a global variable or a PassContext option which can be initialized from the env should a driver script want, rather than forcing this control to be in the environment?

Reply:
No problem, I change the way from using the environment variable to the passContext config. :)